index.html

```diff
@@ -228,29 +303,31 @@ <h3>Profesional Art Printing Data</h3>
                             </li>
                         </ul>
                     </div>
-                    <button type="submit">See Project</button>
+                    <button type="submit" class="card-btn">See Project</button>
 
                 </div>
             </div>
 
-        </div>
 
+        </div>
     </section>
     <section>
         <div class="about">
             <div class="about-header">
-                <h2>About me</h2>
-                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
-                    of
-                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
-                    to
-                    contact me.</p>
-                <button type="submit">Get My Resume</button>
+                <div class="about-content">
+                    <h2>About me</h2>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through
+                        some
+                        of
+                        my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate
+                        to
+                        contact me.</p>
+                    <button type="submit" class="primary-btn">Get My Resume</button>
+                </div>
+
+                <div class="line-dark"></div>
```

Comment:
I notice that the text and illustration are overlapping on this breakpoint, I suggest you fix this as it might make it difficult for users to read the text which will offer a bad user experience.


![Uploading overlap.png…]()


TIP: kindly manipulate the width of the illustration until you get the desired result.